source/_integrations/imgw_pib.markdown

```diff
@@ -15,15 +15,24 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-IMGW-PIB integration uses hydrological data from [Institute of Meteorology and Water Management - National Research Institute](https://hydro.imgw.pl) to present information about rivers and water reservoirs in Poland.
+IMGW-PIB {% term integration %} uses hydrological data from [Institute of Meteorology and Water Management - National Research Institute](https://hydro.imgw.pl) to present information about rivers and water reservoirs in Poland.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Hydrological station:
+    description: "Select a hydrological station from the list of available stations."
+{% endconfiguration_basic %}
+
+## Removing the integration
+
+This integration follows standard integration removal, no extra steps are required.
+
+{% include integrations/remove_device_service.md %}
```

Comment:
Can you move this to be the last section?

Reply:
Done